teams/teams-ps/teams/Set-CsTenantFederationConfiguration.md

```diff
@@ -439,6 +448,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ExternalAccessWithTrialTenants
+When set to 'Blocked', all external access with Teams subscriptions that are all trial will be blocked. Users from trial tenants will not be able to reach to users from tenants that have enabled the Block and vice-versa. The setting will also remove trial tenant users from existing chats if the block gets enabled.
+The default value of the setting is Blocked.
+
+Allowed - Communication with other tenants is allowed
```

Comment:
Allowed - Communication with other tenants is allowed based on other settings.

Reply:
Updated.